public/assets/data/apps.js

```diff
@@ -10,6 +10,13 @@ var apps = [
       "url": "https://www.google.com/?safe=active&ssui=on",
       'image': 'https://img.freepik.com/free-icon/search_318-265146.jpg'
       
+    },
+       {
+      "id": "wikipedia",
+      "title": "Wikipedia",
+      "url": "https://www.wikipedia.org/",
```

Comment:
Maybe use https://en.wikipedia.org/wiki/Main_Page to get to the true home page of wikipedia?

Reply:
> Maybe use https://en.wikipedia.org/wiki/Main_Page to get to the true home page of wikipedia?

there is more then the normal english wiki